app/assets/v2/js/pages/bounty_details.js

```diff
@@ -184,72 +184,85 @@ var isBountyOwner = function(result) {
 }
 
 var pendingChangesWarning = function(issueURL, last_modified_time_remote, now){
-        console.log("checking this issue for updates:");
-        console.log(issueURL);
-        //setup callbacks
-        var changes_synced_callback = function(){
-            document.location.href = document.location.href;
-            //check_for_bounty_changed_updates_REST();
-        };
-        var check_for_bounty_changed_updates_REST = function(){
-            var uri = '/api/v0.1/bounties?github_url='+issueURL;
-             $.get(uri, function(results){
-                results = sanitizeAPIResults(results);
-                var result = results[0];
-                // if remote entry has been modified, refresh the page.  if not, try again
-                if(typeof result == 'undefined' || result['modified_on'] == last_modified_time_remote){
-                    setTimeout(check_for_bounty_changed_updates_REST,2000);
-                } else {
-                    changes_synced_callback();
-                }
-             });
-        };
-        var check_for_bounty_changed_updates_web3 = function(){
-            callFunctionWhenTransactionMined(localStorage['txid'],function(){
-                var bounty = web3.eth.contract(bounty_abi).at(bounty_address());
-                setTimeout(function(){
-                    bounty.bountydetails.call(issueURL, function(error, result){
-                        if(error){
-                            setTimeout(check_for_bounty_changed_updates_web3, 1000);
-                            console.error(error);
+    console.log("checking this issue for updates:");
+    console.log(issueURL);
+
+    //setup callbacks
```

Comment:
its not quite clear to me what changed here.. spacing?

Reply:
Yeah @owocki, they're just spacing issues here